src/components/profile/ProfileCard.tsx

```diff
@@ -56,6 +56,14 @@ function trimNip05(nip05Id: string | undefined) {
   return nip05Id;
 }
 
+async function copyText(text: string) {
+  try {
+    await navigator.clipboard.writeText(text);
+  } catch (err) {
+    console.error("Failed to copy: ", err);
```

Comment:
Should we leave a TODO comment to eventually surface this error to the UI as a toast?

Reply:
we can probably just put in an issue

Reply:
nvm it's handled